src/validate.ts

```diff
@@ -0,0 +1,67 @@
+import * as crypto from 'crypto';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import * as core from '@actions/core';
+import * as openpgp from 'openpgp';
+import * as fetch from 'node-fetch';
+
+import {
+  BASEURL,
+  getUploaderName,
+  setFailure,
+} from './helpers';
+
+const verify = async (platform, filename) => {
+  try {
+    // Read in public key
+    const publicKeyArmored = await fs.readFileSync(
```

Comment:
```suggestion
    const publicKeyArmored = await fs.readFile(
```

Reply:
I went with `*Sync` in a lot of these cases because I _want_ blocking, there's nothing else the CLI can do without those results. 
Interested in the value of async operations here.

Reply:
er, wait. wrong repo :)